test/jobs/__snapshots__/create-group-version-branch.js.snap

```diff
@@ -6,9 +6,9 @@ Object {
   "body": "
 ## There have been updates to the *pouchdb* monorepo: 
 
-- The \`dependency\` [pouchdb](https://www.npmjs.com/package/pouchdb) was updated from \`1.0.0\` to \`2.0.0\`.
-- The \`dependency\` [pouchdb-adapter-utils](https://www.npmjs.com/package/pouchdb-adapter-utils) was updated from \`1.0.0\` to \`2.0.0\`.
-- The \`devDependency\` [pouchdb-core](https://www.npmjs.com/package/pouchdb-core) was updated from \`1.0.0\` to \`2.0.0\`.
+- The \`dependency\` [pouchdb](https://github.com/dep/dep) was updated from \`1.0.0\` to \`2.0.0\`.
```

Comment:
this change doesn't seem right

Reply:
I was wondering about this too though, did you figure out what it was?